ci-operator/step-registry/ipi/conf/networking/ipi-conf-networking-commands.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG=/tmp/secret/install-config.yaml
+
+cluster_network_type=
```

Comment:
@wking given below, do we even need this block? Wouldn't that info come from the variant instead of cluster-network-type?

Reply:
> ... do we even need this block?

If you can set everything through the variant then great, but we still have some consumers (e.g. [here][1]) that use `CLUSTER_NETWORK_TYPE=OVNKubernetes` instead of setting `ovn` in `CLUSTER_VARIANT`.  Is it time to drop that approach?

[1]: https://github.com/openshift/release/blob/be3107193ae1f893a440ec19f78f0151559f00b8/ci-operator/config/openshift-priv/cluster-network-operator/openshift-priv-cluster-network-operator-master.yaml#L48

Reply:
@wking yes, if we can do this without breaking existing things, let's try to move to CLUSTER_VARIANT instead of CLUSTER_NETWORK_TYPE.

Reply:
@wking still need to munge the install-config to set other options before install (like openshift-sdn's multitenant vs. networkpolicy vs. singletenant) but those are much-less-often used and could be separate steps somewhere.

Reply:
> let's try to move to CLUSTER_VARIANT instead of CLUSTER_NETWORK_TYPE.

Spun off into an orthogonal #7628.  I'll rebase whichever after one of the PRs lands.

Reply:
No longer relevant since [this pivot][1].  Marking the thread resolved.

[1]: https://github.com/openshift/release/pull/7625#issuecomment-598875585